WooCommerce/Classes/ViewModels/CardPresentPayments/CardPresentModalError.swift

```diff
@@ -12,7 +12,8 @@ final class CardPresentModalError: CardPresentPaymentsModalViewModel {
     /// A closure to execute when the primary button is tapped
     private let primaryAction: () -> Void
 
-    let mode: PaymentsModalMode = .oneActionButton
+    let textMode: PaymentsModalTextMode = .noBottomInfo
```

Comment:
Splitting the "mode" in two enumerations. Labels and buttons are kind of orthogonal, and this was brought up [in a previous PR](https://github.com/woocommerce/woocommerce-ios/pull/4065#discussion_r622610207)